src/Compilers/CSharp/Test/Semantic/Semantics/AmbiguousOverrideTests.cs

```diff
@@ -227,13 +219,12 @@ class Base2<A, B> : Base<A, B>
 }
 class Derived : Base2<int, int>
 {
-    public override void Method(ref List<int> a, out List<int> b) { b = null; } // Reports warning about runtime ambiguity
+    public override void Method(ref List<int> a, out List<int> b) { b = null; } // No warning - the compiler produces a methodimpl record to disambiguate
     public override void Method(ref List<int> a) { } // No warning when ambiguous signatures are spread across multiple base types
 }";
 
             CreateCompilation(source).VerifyDiagnostics(
-                // (5,25): warning CS1957: Member 'Derived.Method(ref System.Collections.Generic.List<int>, out System.Collections.Generic.List<int>)' overrides 'Base<int, int>.Method(ref System.Collections.Generic.List<int>, out System.Collections.Generic.List<int>)'. There are multiple override candidates at run-time. It is implementation dependent which method will be called.
-                Diagnostic(ErrorCode.WRN_MultipleRuntimeOverrideMatches, "Method").WithArguments("Base<int, int>.Method(ref System.Collections.Generic.List<int>, out System.Collections.Generic.List<int>)", "Derived.Method(ref System.Collections.Generic.List<int>, out System.Collections.Generic.List<int>)"));
```

Comment:
>WRN_MultipleRuntimeOverrideMatches [](start = 37, length = 34)

Same comment as for the other tests affected by the removal of WRN_MultipleRuntimeOverrideMatches

Reply:
That is tested in `CodeGenOverridingAndHiding`.

---
In reply to: [442519288](https://github.com/dotnet/roslyn/pull/44025#discussion_r442519288) [](ancestors = 442519288)